Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -971,8 +972,9 @@ def main(argv):
     header_set = set()
     file_extension = argv[1]
     for i in range(3, len(argv)):
-        with open(argv[2] + argv[i]) as file:
-            new_types, new_enums, new_headers, new_typedefs = parse_serialized_types(file, argv[i])
+        path = os.path.join(os.getcwd(), argv[i]) if i == (len(argv) - 1) else argv[2] + argv[i]
```

Comment:
Is it possible to have create-http-header-name-table generate a .serialization.in file as well, include it from derived sources into the private headers, then include $(WebCorePrivateHeaders)/HTTPHeaderNames.serialization.in in the definition of HTTPHeaderNames.serialization.in?  You might have to put $(WebKit2)/ before all the existing paths.  I think that would be cleaner and heading in a better direction